### ISSUE TITLE
Improve colour contrast for highlighted code blocks

### ISSUE THREAD

**Issue by nchlswhttkr** [nchlswhttkr/website#27]
It would be nice for code blocks with highlighting to all meet AA contrast requirements.

Most of the [Chroma themes](https://gohugo.io/content-management/syntax-highlighting/) have trouble in one more languages (like comments and operators) or don't cover all the languages I need (colourless git diffs).

I figure the best route to go for this is to use a copy an existing theme into my own CSS file (`hugo gen chromastyles --style=vs > assets/highlight.css `), and add some logic to only load it on pages that need highlighting.

I'm not concerned about having differing light/dark versions, so I only need a single theme.

```yaml
// config.yml
markup:
  highlight:
    noClasses: false
```

```html
{{ if in .Content "class=\"highlight\"" }}
<link rel="stylesheet" type="text/css" href="{{ (resources.Get "highlight.css" | minify ).Permalink }}" />
{{ end }}
```

**Comment by nchlswhttkr** [comment]
Got it [working and ready](https://github.com/nchlswhttkr/website/compare/master...accessible-code-highlighting), will merge at some point in the near future.

It would be nice to strip out empty CSS rules when minifying, but it's not an immediate concern. Could be done at the tooling level (https://github.com/tdewolff/minify) or as part of the build script (easier).

**Comment by nchlswhttkr** [comment]
Empty rules are now removed in the build script!

**Comment by nchlswhttkr** [comment]
Implemented with 4769145
